src/igvfd/tests/test_schema_primary_cell.py

```diff
@@ -91,3 +91,23 @@ def test_sorted_fraction_detail_dependency(testapp, primary_cell):
         {'sorted_fraction': primary_cell['@id'],
          'sorted_fraction_detail': 'I am a sorted fraction detail.'})
     assert res.status_code == 200
+
+
+def test_patch_virtual(primary_cell, testapp):
+    res = testapp.patch_json(
+        primary_cell['@id'],
+        {'virtual': True})
+    assert res.status_code == 200
+    res = testapp.patch_json(
+        primary_cell['@id'],
+        {'virtual': False})
+    assert res.status_code == 200
+    res = testapp.patch_json(
+        primary_cell['@id'],
+        {'virtual': ''}, expect_errors=True)
+    assert res.status_code == 422
+
+
+def test_read_virtual(primary_cell_v7, testapp):
```

Comment:
i don't think we need this test

Reply:
This test was to confirm that the default value for the schema property virtual was working correctly.  The test has been removed.